File: src/main/java/net/allape/bus/Windows.java
package net.allape.bus;

import com.intellij.ui.content.Content;
import net.allape.window.explorer.XFTPExplorerWindow;

import java.util.HashMap;
import java.util.Map;

public final class Windows {

    /**
     * 默认的窗口名称
     */
    public static final String WINDOW_DEFAULT_NAME = "Explorer";

    // 当前打开的窗口
    public static final Map<Content, XFTPExplorerWindow> windows = new HashMap<>(10);

}

File: src/main/kotlin/net/allape/common/Windows.kt
package net.allape.common

import com.intellij.ui.content.Content
import net.allape.xftp.ExplorerWindow

class Windows {

    companion object {

        // 默认的窗口名称
        const val WINDOW_DEFAULT_NAME = "Explorer"

        // 当前打开的窗口
        val windows: HashMap<Content, ExplorerWindow> = HashMap(10)

    }

}